lib/systems/inspect.nix

```diff
@@ -59,7 +59,7 @@ rec {
 
     isBSD          = { kernel = { families = { inherit (kernelFamilies) bsd; }; }; };
     isDarwin       = { kernel = { families = { inherit (kernelFamilies) darwin; }; }; };
-    isUnix         = [ isBSD isDarwin isLinux isSunOS isCygwin isRedox ];
+    isUnix         = [ isBSD isDarwin isLinux isSunOS isCygwin isRedox isEmscripten ];
```

Comment:
I highly doubt this is unix

Reply:
It is!

> It even emulates an entire [POSIX](https://en.wikipedia.org/wiki/POSIX) operating system, enabling programmers to use functions from the [C standard library](https://en.wikipedia.org/wiki/C_standard_library) (libc).

https://en.wikipedia.org/wiki/Emscripten